libraries/ESP32/examples/DeepSleep/TouchWakeUp/TouchWakeUp.ino

```diff
@@ -11,7 +11,13 @@ Author:
 Pranav Cherukupalli <cherukupallip@gmail.com>
 */
 
-#define Threshold 40 /* Greater the value, more the sensitivity */
+#if CONFIG_IDF_TARGET_ESP32
+  #define THRESHOLD 40      /* Greater the value, more the sensitivity */
+#elif CONFIG_IDF_TARGET_ESP32S2
+  #define THRESHOLD   30000 /* Lower the value, more the sensitivity */
+#else //CONFIG_IDF_TARGET_ESP32S3 + default for other chips (to be adjusted) */
+  #define THRESHOLD   80000 /* Lower the value, more the sensitivity */
+#endif
```

Comment:
I have tested it and it works better with this setup:

ESP32 -> Threshold = 40
S2/S3 -> Threshold = 1000

Reply:
For ESP32 we do have threshold 40.
But for S2/S3 it cannot be that small. Normal reading is higher that this value.
For S2 if the pad is touched, reading from touchRead is above 45k. For S3 its above 95k

Reply:
I think that this is actually the delta, not the reading value itself. 
It worked better (faster when touching the S2/S3 pad) when I changed it to 1,000
You can try it to check it.

Reply:
I have tested that a bit now, and managed to change it to 5000. That worked the best for me.
The 1000 was too much sensitive. I have attached the Dupont cable to the pin and it triggered even if I touched the rubber part of the cable, not the wire.
The 5000 triggers immediately I touch the wire :)